source/src/ble/cs_BleCentral.cpp

```diff
@@ -28,51 +28,37 @@ void handle_discovery(ble_db_discovery_evt_t* event) {
 }
 
 #if NORDIC_SDK_VERSION == 15
-BleCentral::BleCentral():
-	_discoveryModule {
-		.discovery_in_progress = false,
-		.discovery_pending = false,
-		.conn_handle = BLE_CONN_HANDLE_INVALID
-	}
-{
+BleCentral::BleCentral() {
+	_discoveryModule.discovery_in_progress = false;
+	_discoveryModule.discovery_pending     = false;
+	_discoveryModule.conn_handle           = BLE_CONN_HANDLE_INVALID;
 }
 #else
 
 // TODO: Get rid of this macro and only create data structure if class instance is used...
-NRF_BLE_GQ_DEF(m_ble_gatt_queue,
-		NRF_SDH_BLE_PERIPHERAL_LINK_COUNT,
-		NRF_BLE_GQ_QUEUE_SIZE);
-
-BleCentral::BleCentral():
-	_discoveryModule {
-		.discovery_in_progress = false,
-		.conn_handle = BLE_CONN_HANDLE_INVALID
-	},
-	_discoveryInit {
-		.evt_handler = NULL,
-		.p_gatt_queue = NULL
-	}
```

Comment:
What happened to this init?

Reply:
`_discoveryInit` has more fields like this, so this shouldn't compile with an up to date SDK.